src/main/java/Duke.java

```diff
@@ -1,10 +1,159 @@
+
+                        for (int i = seq + 1; i < input.length; i++) {
+                            timing = timing + " " + input[i];
+                        }
+
+                        itemList.add(new Deadline(joined, timing.trim()));
+                        fileContents.add(new Deadline(joined, timing.trim()).toString());
+                        count++;
+                        System.out.println("Added " + joined + "\nYou now have " + count + " items in your list.");
+                        break;
+                    case "event":
+                        int seq2 = 0;
+                        while (!input[seq2].equals("/at")) {
+                            joined = joined + " " + input[seq2];
+                            seq2++;
+                            if (seq2 == input.length) {
+                                throw new DukeDescriptionException(input[0]);
+                            }
+                        }
+
+                        if (seq2 + 1 == input.length) {
+                            throw new DukeTimingException(input[0]);
+                        }
+
+                        for (int i = seq2 + 1; i < input.length; i++) {
+                            timing = timing + " " + input[i];
+                        }
+                        itemList.add(new Event(joined, timing.trim()));
+                        fileContents.add(new Event(joined, timing.trim()).toString());
+                        count++;
+                        System.out.println("Added " + joined + "\nYou now have " + count + " items in your list.");
+                        break;
+                    case "delete":
+                        count--;
+                        itemList.remove(Integer.parseInt(input[1]));
+                        fileContents.remove(Integer.parseInt(input[1]));
+                        System.out.println("I have removed item " + input[1] + ".");
+                    default:
+                        throw new DukeNotFoundException();
+                }
+            }
+            System.out.println(h_rule);
+            PrintWriter writer = new PrintWriter(log);
+            for (String line : fileContents) {
+                writer.println(line);
+            }
+            writer.close();
+        }
+        sc.close();
+        return 0;
+    }
+
     public static void main(String[] args) {
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Greetings! I am Duke! How may I assist you?\n" + h_rule);
+        int res = -1;
+        File logFile = new File("./logs");
+
+        try {
+            if (logFile.isFile()) {
+                Scanner logs = new Scanner(logFile);
+                while (logs.hasNextLine()) {
```

Comment:
Would it be better to have a gap between the scanner statement and the while loop statement? It might make it easier to read. I noticed this in a few other places too.